src/components/build/BuildInformation.test.js

```diff
@@ -55,11 +55,11 @@ describe('BuildInformation', () => {
     });
   });
 
-  describe('BuildDownload', () => {
-    it('should match previous snapshot', () => {
-      const BuildDownload = wrapper.find('BuildDownload');
-
-      expect(BuildDownload).toMatchSnapshot();
-    });
-  });
+  // describe('BuildDownload', () => {
```

Comment:
I think this block should be completely removed instead of just commented out.

Reply:
👍

Reply:
The idea was to leave it there to be reintroduced in the future. However, since that page is not used anymore, probably it would be ok to remove it